Bakermans changes

BakermanLP opened the pull request:
Einige kleine Änderungen, die einzeln in Commits dokumentiert sind.

BakermanLP committed:
Damit beim kopieren auf den Server die Friends Liste nicht
ueberschrieben wird.

BakermanLP committed:
Support fuer MinefactoryReloaded aus ThermalFoundaition geloescht

BakermanLP committed:
StorageDrawers Compacting Storage kompatibel zu Sand,
Compressed Sand ...

BakermanLP committed:
Thaumcraft Erze kommen aus ExNihilo und koennen hier
abgeschaltet werden

BakermanLP committed:
Update Checker ausschalten (Startvorgang beschleunigen) und World
Generation ausschalten mittels Blacklist Eintrag

DerOliDE commented:
Pull lasse ich erst einmal offen bis mit 0.6.1 die Server-Problematik behoben wurde.

BakermanLP committed:
Ein paar mehr Möglichkeiten mit Storage Drawern

BakermanLP committed:
Questtexte fuer Kapitel eins in das JSON File

Leider hatte das konvertieren ins HQM bei mir nicht geklappt.

BakermanLP committed:
fix combak/ShatteredWorld#95 Alu Eimer

BakermanLP committed:
fix combak/ShatteredWorld#36 : Stone Slab wird von Forest Day überlagert

BakermanLP committed:
fix combak/ShatteredWorld#38 : für Sky Stone Dust gibt es jetzt keine
Rezepte mehr

BakermanLP committed:
fix combak/ShatteredWorld#96 : Pams pot und saucepan nur in der Schmelze
ref combak/ShatteredWorld#102 : TConstruct Smeltery "realistischer"

Leider gestaltet sich der Teil mit Cobalt usw deutlich schwerer, da hier
eine Unmenge an schmelzbaren Materialen exisitert. Deswegen hier nur die
Anpassung von Redstone auf 3000°C.

BakermanLP commented:
Könntest du den mal "langsam" mergen, sonst wird das komplett unübersichtlich. Die Änderungen in TinkerConstruct.zs kommen von meiner etwas älteren Version vor 0.6.1. Ich habe da einfach die 0.6.1er Version reinkopiert und dann die Änderungen gemacht.

BakermanLP committed:
Kleine Bugfixes und ein kleiner Gag

DerOliDE commented:
Ich merge das erst wenn das Problem mit den Custom Fluids aus der Welt ist und ein automatisches merge ist schon gar nicht mehr möglich.

DerOliDE commented:
P.S.: Fluid Problem wird heute noch beendet und an Deine Changes mache ich mich dann morgen.

DerOliDE commented:
So alles manuell übernommen nur was mit der cofh friends.cfg gemacht wurde ist mir nicht ersichtlich, ich vermute mal irgendwelche Lese-/Schreiberechte wurden verändert.

BakermanLP commented:
Die friends.cfg habe ich gelöscht. Damit wird sie beim update nicht
überschrieben

Oliver notifications@github.com schrieb am Mo., 22. Juni 2015 15:55:

> Closed #91 https://github.com/combak/ShatteredWorld/pull/91.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/combak/ShatteredWorld/pull/91#event-336723792.

DerOliDE commented:
Okay dann ist jetzt alles erledigt.
